cluster-service/Makefile

```diff
@@ -4,10 +4,11 @@ deploy:
 	oc process --local -f deploy/openshift-templates/arohcp-namespace-template.yml \
 	  -p ISTIO_VERSION=asm-1-20 | oc apply -f -
 	oc process --local -f deploy/openshift-templates/arohcp-db-template.yml | oc apply -f -
-	oc process --local -f deploy/openshift-templates/arohcp-secrets-template.yml | oc apply -f -
+	oc process --local -f deploy/openshift-templates/arohcp-secrets-template.yml \
+	  -p PROVISION_SHARDS_CONFIG="cHJvdmlzaW9uX3NoYXJkczoKLSBpZDogMQogIGh5cGVyc2hpZnRfY29uZmlnOiB8CiAgICBhcGlWZXJzaW9uOiB2MQogICAga2luZDogQ29uZmlnCiAgICBjbHVzdGVyczoKICAgIC0gbmFtZTogZGVmYXVsdAogICAgICBjbHVzdGVyOgogICAgICAgIHNlcnZlcjogaHR0cDovL2xvY2FsaG9zdAogICAgdXNlcnM6CiAgICAtIG5hbWU6IGRlZmF1bHQKICAgICAgdXNlcjoKICAgICAgICB0b2tlbjogZm9vCiAgICBjb250ZXh0czoKICAgIC0gbmFtZTogZGVmYXVsdAogICAgICBjb250ZXh0OgogICAgICAgIGNsdXN0ZXI6IGRlZmF1bHQKICAgICAgICB1c2VyOiBkZWZhdWx0CiAgICBjdXJyZW50LWNvbnRleHQ6IGRlZmF1bHQKICBzdGF0dXM6IGFjdGl2ZQogIHJlZ2lvbjogZWFzdHVzCiAgY2xvdWRfcHJvdmlkZXI6IGF6dXJlCiAgdG9wb2xvZ3k6IGRlZGljYXRlZAo=" | oc apply -f -
```

Comment:
this is a fake provisioning shard to fix CS crashlooping. we can start providing real shards in about 1-2 weeks when CS offers config options for maestro backed provisioning shards

Reply:
Do we want to store this as a `provision_shards.config` file in this repo and base64 encode it in the Makefile?

I think we will want to adjust the `region: eastus` part of the config --> westus3 as well

Reply:
sure i can pull it out into a file

Reply:
done